### PR TITLE
ref(storybook): Burn it all down

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Storybook will be back in some capacity in the future. But for now, this makes our lives much easier not having it pull in literally 4000 lines of extra yarn.lock dependencies.

### CODE REVIEW

**Review by vuluongj20 — APPROVED** [review]

**Review by scttcper — APPROVED** [review]
🔥❤️‍🔥

**Review by ryan953 — APPROVED** [review]

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -119,19 +119,12 @@ jobs:
...
-        run: |
-          yarn storybook-build
-
   webpack:
     if: github.ref == 'refs/heads/master' || needs.files-changed.outputs.frontend == 'true'
     needs: files-changed
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8  # v3.1.0
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
```
Comment (evanpurkhiser) [inline .github/workflows/frontend.yml:127]: damnit

File: package.json [hunk 2]
```diff
@@ -209,30 +197,22 @@
...
     "webpack": "5.75.0",
-    "css-loader": "^5.2.6",
-    "dotenv-webpack": "^7.1.0",
-    "html-webpack-plugin": "^5.0.0",
-    "less-loader": "^11.0.0",
-    "style-loader": "^3.3.1",
-    "terser-webpack-plugin": "^5.1.4",
-    "webpack-virtual-modules": "^0.4.3",
-    "colors": "1.4.0"
+    "style-loader": "^3.3.1"
```
Comment (evanpurkhiser) [inline package.json:215]: This is what I am here for god damn yes

File: package.json [hunk 3]
```diff
@@ -209,30 +197,22 @@
...
     "react-test-renderer": "17.0.2",
     "size-limit": "^5.0.5",
-    "storybook-dark-mode": "^1.1.2",
     "stylelint": "14.8.2",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-recommended": "^8.0.0",
     "tocbot": "^4.20.0",
     "tsconfig-paths": "^4.1.1",
     "typescript-styled-plugin": "^0.18.2",
+    "colors": "1.4.0",
```
Comment (evanpurkhiser) [inline package.json:206]: We weren't actually directly depending on colors lol

**Review by mitsuhiko — APPROVED** [review]

**Comment by mitsuhiko** [comment]
🫗📖

**Review by billyvg — APPROVED** [review]
Can we at the minimum have a plan for the documented parts of our design system that do exist?

Colors and icons are also helpful imo.

**Comment by evanpurkhiser** [comment]
I've disconnected https://storybook.sentry.dev from the git repo. That won't be going anywhere.

I believe @vuluongj20 already has a plan in place for beginning some work on a design system site.

**Pull request merged**